dotnet/src/InternalUtilities/samples/InternalUtilities/BaseTest.cs

```diff
@@ -110,7 +113,10 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
             if (request.Content is not null)
             {
                 var content = await request.Content.ReadAsStringAsync(cancellationToken);
-                this._output.WriteLine(content);
+                string formattedContent = JsonSerializer.Serialize(JsonSerializer.Deserialize<JsonElement>(content), s_jsonSerializerOptions);
```

Comment:
Consider wrapping this in a try catch, in case the content isn't json, and just writing the original content in case of failure.